### DISCUSSION TITLE
More information about "class" for each record

### DISCUSSION THREAD

**Discussion by Shaima3j** [petrobras/3W#72] [Q&A] [answered]
Good day,

I have a question about the class. Each record has a class ranging from 0 to 8, how each row of data was classified?
What is the mathematical reason for a record having class 1 for example? What kind of calculation was performed on the tag values?

Thank you.

**Comment by utkuaslan85** [comment]
Hi,
Classes shows the event type. Dataset.ini file shows their labeling.
[https://github.com/petrobras/3W/blob/main/dataset/dataset.ini](https://github.com/petrobras/3W/blob/main/dataset/dataset.ini)

Some of the events have transient states. For example in event type 1, an instance
- starts in normal state which is Class 0
- changes to transient state which is class 101
- and finally reaches to faulty steady-state which is 1

I hope it is clear now.

**Comment by ricardoevvargas** [comment, marked as answer]
Hi, @Shaima3j. 

I believe @utkuaslan85's comment clarifies your questions. If you and others have other questions, they are welcome here. We can reopen this discussion.

By the way, the labeling of each real instance row was done by human experts with supporting software that displays multiple time series. 
Each row of simulated instances was automatically labeled as a function of simulation planning. Each line of hand-drawn instances was automatically labeled according to drawings generated by human experts. The numbers associated with labels are just codes. Therefore, there is no mathematical meaning in them. Labeling does not involve specific mathematical calculations, but interpretation of the behavior of variables (increase, maintenance or reduction of values).

**Comment by JohnIsak** [comment]
In a similar manner I am in need of more information regarding the "state" feature. As far as I see in dataset.ini "state" refers to "Well operational status". The subset of the data I am currently working on contain these values for state: 	[<NA>, 0, 1, 7, 8, 4, 5, 3, 6]. I do however have no idea about how to interpret the meaning of these different values.
